src/tlo/methods/hiv.py

```diff
@@ -40,7 +40,7 @@
 from tlo.util import create_age_range_lookup
 
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+logger.setLevel(logging.CRITICAL )
```

Comment:
Any reason you are setting this to critical? I think if we don't want `.INFO` logs from Hiv module, we can configure cervical cancer analyses to only allow logging.INFO from cervical cancer module